korman/properties/modifiers/water.py

```diff
@@ -471,3 +471,36 @@ class PlasmaWaveTexState(PlasmaWaveState, PlasmaModifierProperties):
     def export(self, exporter, bo, so):
         waveset = exporter.mgr.find_create_object(plWaveSet7, name=bo.name, so=so)
         self.convert_wavestate(waveset.state.texState)
+
+
+class PlasmaBuoyObject(idprops.IDPropObjectMixin, bpy.types.PropertyGroup):
+    display_name = StringProperty(name="Display Name")
```

Comment:
I've been migrating toward having these lists display the actual name of the object or "[Empty]" if unset. IMO, it makes things less tedious.